frontend/src/app/globals.css

```
@@ -49,40 +49,46 @@
 }
 
 :root {
-  --font-naruon-sans: Pretendard, "Apple SD Gothic Neo", "Malgun Gothic", "Segoe UI", ui-sans-serif, system-ui, sans-serif;
+  --font-naruon-sans: Pretendard, SUIT, "Noto Sans KR", "Apple SD Gothic Neo", "Malgun Gothic", "Segoe UI", ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)